### DISCUSSION TITLE
运行错误

### DISCUSSION THREAD

**Discussion by killLuoti** [1715173329/UnblockNeteaseMusic#492] [Q&A]
INFO: (app) HTTP Server running @ http://0.0.0.0:8088
INFO: (app) HTTPS Server running @ http://0.0.0.0:8089
ERROR: (hook) Promise.any is not a function
    TypeError: Promise.any is not a function
        at match (/root/UnblockNeteaseMusic/src/provider/match.js:64:34)
        at processTicksAndRejections (internal/process/task_queues.js:95:5)
        at async Promise.all (index 0)
INFO: (spawn) running yt-dlp -f 140 --dump-json ytsearch1:十年 - 陈奕迅
INFO: (spawn) running yt-dlp -f 140 --dump-json https://www.youtube.com/watch?v=7OdargV9K88
ERROR: (hook) Promise.any is not a function
    TypeError: Promise.any is not a function
        at match (/root/UnblockNeteaseMusic/src/provider/match.js:64:34)
        at processTicksAndRejections (internal/process/task_queues.js:95:5)
        at async Promise.all (index 0)
INFO: (spawn) running yt-dlp -f 140 --dump-json ytsearch1:纸船 - 薛之谦 / 郁可唯
INFO: (spawn) running yt-dlp -f 140 --dump-json https://www.youtube.com/watch?v=5XSDQznDIdw
ERROR: (hook) Promise.any is not a function
    TypeError: Promise.any is not a function
        at match (/root/UnblockNeteaseMusic/src/provider/match.js:64:34)
        at processTicksAndRejections (internal/process/task_queues.js:95:5)
        at async Promise.all (index 0)
INFO: (spawn) running yt-dlp -f 140 --dump-json ytsearch1:感官先生 - 刘凤瑶
INFO: (spawn) running yt-dlp -f 140 --dump-json https://www.youtube.com/watch?v=Dii0Vp2lReU

**Comment by killLuoti** [comment]
ERROR: (hook) Promise.any is not a function
    TypeError: Promise.any is not a function
        at match (/root/UnblockNeteaseMusic/src/provider/match.js:64:34)
        at processTicksAndRejections (internal/process/task_queues.js:95:5)
        at async Promise.all (index 0)
(node:30202) UnhandledPromiseRejectionWarning: undefined
(node:30202) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 41)
(node:30202) UnhandledPromiseRejectionWarning: undefined
(node:30202) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 44)
WARN: (request) The request timed out, or the requester didn't handle the response.
    url: "http://kuwo.cn/search/list?key=%E7%94%9F%E5%A6%82%E5%A4%8F%E8%8A%B1%E6%9C%B4%E6%A0%91"

**Comment by killLuoti** [comment]
能找到音源，但是客户端还是只能听一点点，本版为 2.9.7

**Comment by pan93412** [comment]
假如你的 Node.js 不是最新版，請使用 `precompiled/app.js`（或者是 UNM 目錄的 `app.js`）。
